kedro_airflow_k8s/airflow_dag_template.j2

```diff
@@ -33,6 +37,34 @@ with DAG(
         python_callable=start_mlflow_run,
     )
 
+    def create_pipeline_storage(ti, **kwargs):
+        with client.ApiClient(config.load_incluster_config()) as api_client:
+            v1 = client.CoreV1Api(api_client)
+
+            pvc = k8s.V1PersistentVolumeClaim(metadata=k8s.V1ObjectMeta(name=pvc_name, namespace='{{ config.namespace }}'),
+                    spec=k8s.V1PersistentVolumeClaimSpec(access_modes=['ReadWriteOnce'],
```

Comment:
Consider pushing access modes to config - some setups may have for example NFS that allows ReadWriteMany

Reply:
Good point, I am changing this.